.github/workflows/pr-builder.yml

```
@@ -0,0 +1,49 @@
+    branches: [main, master]
+  workflow_dispatch:
+
+env:
+  MAVEN_OPTS: -Xmx4g -Xms1g
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
```

actionlint (1.7.12)

[error] the runner of "actions/checkout@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)